hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -1863,11 +1863,12 @@
             Requires `EnMasking` parameter to be enabled.
             SW randomly configures the KMAC in any hashing mode/strength, and enable EDN mode.
             Randomly enable/disable the `entropy_refresh`.
-            // TODO - error handling is not complete, do not enable `wait_timer` yet.
-            KMAC should send request to EDN once `CmdStart` is written, and should send out another
-            request to EDN when either:
+            Randomly configure `wait_timer` values (zero for disable, non-zero for timer expire).
+            The `wait_timer` should be greater than the expected EDN latency.
+            KMAC should send EDN request after `entropy_ready` is set.
+            KMAC also should send out another request to EDN when either:
```

Comment:
That looks great, thanks Eunchan! Just one comment here -
Because EDN request is an internal signal, do you have any suggestions in terms of how to ensure the EDN is fetched?
For example, I might program the KMAC wrong to use unmasked mode, and ended up not using the EDN feature.

I thought about two solutions:
1). Programming the `wait_timer` to a super small value, and always expect to see the EDN timeout error.
     Then we adjust the `wait_timer` value with the correct timeout value, and keep other configurations the same value. Then check the digest.
2). Or we can probe internal to make sure the EDN is fetched. This will request a SV sequence.

Do you have any better suggestions?

Reply:
Hum.. Good point! I found nothing indicating KMAC fetched EDN entropy correctly. It only reports when it failed to fetch. The approach 1 should be the best bet with current design.

Reply:
great, maybe we can modify the testplan description to reflect that?

Reply:
Do you want to cover that in this testplan or new testplan?

Reply:
I think in this existing testplan is good enough, since the goal is to check if KMAC fetched entropy from EDN. I do not think we need another testplan ;)

Reply:
OK. Let me revise the PR to reflect that

Reply:
Revised. Please take a look @cindychip